README.md

```diff
@@ -63,53 +70,56 @@ _We do not recommend using the default options for PostgreSQL. The above values
 - `DATABASE_HOST`: The IP address Docker is running on. You can find this by running `docker-machine ip` but it's usually `192.168.99.100` instead of `localhost`
 - `DATABASE_URL_DEV`: This will need to be adjusted as well, for example `DATABASE_URL_DEV=postgres://postgres:.@localhost:5432/postgres` would become `DATABASE_URL_DEV=postgres://postgres:.@192.168.99.100:5432/postgres`

```

Comment:
Should these be DATABASE_URL_DEV or DATABASE_URL? Will depend on answers to prior comments.

Reply:
It should be DATABASE_URL. I will fix this in another PR.